src/Tools/IiifHelper.php

```diff
@@ -67,21 +69,31 @@ public function getRemoteIIIFImageInfo($remoteUrl) {
     }
     $message= $this->t('Looks like data fetched from @url is not in IIIF or JSON format.<br> JSON says: @$jsonerror <br>Please check your URL!',
       [
-        '@url' => $remoteUrl,
+        '@url' =>  $infojsonurl,
```

Comment:
One space too much after the => you can always run `phpcs --standard=Drupal` on the file or let PHPStorm do that for you too